### PR TITLE
feat(RangeCalendar): add description and sandbox for RangeCalendar

### PR DESCRIPTION
**Pull request by Arucard89**

### CODE REVIEW

**Comment by Arucard89** [comment]
wait for new version of date-components

**Comment by Arucard89** [comment]
include fix for https://github.com/gravity-ui/landing/issues/166

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@doc-tools/transform": "^3.11.0",
     "@gravity-ui/components": "2.1.0",
-    "@gravity-ui/date-components": "^1.3.0",
+    "@gravity-ui/date-components": "^1.4.0",
```
Comment (imsitnikov) [inline package.json:9]: @Arucard89 here is a problem with displaying the "Size" block:

image

Can we place them in separate blocks?

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Arucard89) [reply package.json:9]: @imsitnikov , hi. I've checked other components and found that the problem is not the only one. Other components have the same problem with positioning of items in row. For Example in Button component:
image
So I want to suggest another decision. What do you think about changing css to wrap element automatically?
https://github.com/gravity-ui/landing/pull/167/commits/f334d5a7139b3df7c4d67e4610ba53c6ee30cb71

Comment (imsitnikov) [reply package.json:9]: This is a good idea, but can we also add `align-items: center;` and `justify-content: center;`?

Comment (Arucard89) [reply package.json:9]: Added: https://github.com/gravity-ui/landing/pull/167/commits/8f71cf414930bfe77d25146c8c7e5d162bf1b46b

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**